glide.yaml

```diff
@@ -34,7 +38,7 @@ import:
 - package: github.com/zach-klippenstein/goregen
 - package: github.com/spf13/cobra
 - package: github.com/spf13/viper
-- package: github.com/Sirupsen/logrus
+- package: github.com/sirupsen/logrus
```

Comment:
needed because of https://github.com/fabric8-services/fabric8-auth/commit/468d4e6d7bb84af300d28a432293bcc758b4e708